src/lib/libemval.js

```diff
@@ -42,16 +42,16 @@ var LibraryEmVal = {
     return emval_handles.length / 2 - {{{ EMVAL_RESERVED_HANDLES }}} - emval_freelist.length;
   },
 
-  _emval_register_symbol__deps: ['$emval_symbols', '$readLatin1String'],
+  _emval_register_symbol__deps: ['$emval_symbols', '$AsciiToString'],
   _emval_register_symbol: (address) => {
-    emval_symbols[address] = readLatin1String(address);
+    emval_symbols[address] = AsciiToString(address);
   },
 
-  $getStringOrSymbol__deps: ['$emval_symbols', '$readLatin1String'],
+  $getStringOrSymbol__deps: ['$emval_symbols', '$AsciiToString'],
   $getStringOrSymbol: (address) => {
     var symbol = emval_symbols[address];
     if (symbol === undefined) {
-      return readLatin1String(address);
+      return AsciiToString(address);
```

Comment:
Why does this function start with a capital letter .. so strange.    Should we add make a lower case version and deprate this one?

Reply:
Idk, but it was always like that, I believe it existed from early days of Emscripten.